dblock.go

```diff
@@ -155,3 +155,18 @@ func GetDBlockHead() (string, error) {
 
 	return head.KeyMR, nil
 }
+
+// ReplayDBlockFromHeight requests DBlock states to be emitted over the LiveFeed API
+func ReplayDBlockFromHeight(startheight int64, endheight int64) (*JSON2Response, error) {
```

Comment:
Is there a reason we are returning the `JSON2Response` vs a type a user can more easily work with?

I'm guessing the response is something as simple as `success`?
We should really return the unmarshalled type here, and not have the caller do that work